plugins/generator-1.18.2/forge-1.18.2/templates/gui/gui_container.java.ftl

```diff
@@ -114,7 +114,7 @@ public class ${name}Menu extends AbstractContainerMenu implements Supplier<Map<I
 					<#if hasProcedure(component.disablePickup) || component.disablePickup.getFixedValue()>
 						@Override public boolean mayPickup(Player entity) {
 							<#if hasProcedure(component.disablePickup)>
-								return !<@procedureOBJToLogicCode component.disablePickup/>;
+								return !<@procedureOBJToConditionCode component.disablePickup "false"/>;

```

Comment:
Can't this construction be inlined (since currently `hasProcedure(component.disablePickup)` is being called twice at virtually the same place)?

Reply:
You are right. However, as the conditions need to be inverted (so the UI is easier to understand and use for users), I added another argument to invert the condition code (not the default value).